bucket/retroshare.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.6.5",
+    "description": "RetroShare is a Free and Open Source cross-platform, Friend-2-Friend and secure decentralised communication platform.",
+    "homepage": "https://retroshare.cc/",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/RetroShare/RetroShare/releases/download/v0.6.5/RetroShare-0.6.5-Windows-Portable-20190204-1-gfb005f041-Qt-5.12.0.7z",
+    "hash": "a1790e72c4400b2a001fb4b0d369be883e5846e2c23fa06af148a161d14f0fa8",
+    "bin": [
+        "retroshare.exe",
+        "retroshare-nogui.exe",
+        [
+            "retroshare-nogui.exe",
+            "retroshare-webui",
+            "--webinterface 9090",
+            "--docroot \"$dir\\webui\""
+        ]
+    ],
+    "shortcuts": [
+        [
+            "retroshare.exe",
+            "RetroShare"
+        ]
+    ],
+    "persist": "Data",
+    "checkver": {
+        "url": "https://retroshare.cc/downloads.html",
+        "regex": "RetroShare-(?<version>[\\d.]+)-Windows-Portable-(?<build>[\\da-zA-Z\\-.]+).7z\">Retroshare \\(Portable\\)"
```

Comment:
```suggestion
        "regex": "RetroShare-([\\d.]+)-Windows-Portable-(?<build>[\\w-.]+).7z\">Retroshare \\(Portable\\)"
```

Reply:
I find there are tons manifest use `?<version>` :joy:

Reply:
It is irrelevant when it is firts captured group.

Reply:
Yeah, so those manifests make me confused. There is even such manifest. :joy:  https://github.com/lukesampson/scoop-extras/blob/a84b257fd9636d02295b48c3fd32826487ca9bd3/bucket/graphicsgale.json#L21

Reply:
This could be for historical reason.